apps/console/src/features/email-and-sms/pages/email-and-sms.tsx

```diff
@@ -74,73 +62,65 @@ export const EmailAndSMSPage: FunctionComponent<EmailAndSMSPageInterface> = (
         history.push(AppConstants.getPaths().get("EMAIL_PROVIDER"));
     };
 
-    /**
-     * Get the page details based on the enabled providers.
-     */
-    const getPageDetails = (): {
-        description: string;
-        pageTitle: string;
-        title: string;
-    } => {
-        if ((isEmailProviderEnabled && isSMSProviderEnabled) || (!isEmailProviderEnabled && !isSMSProviderEnabled)) {
-            return {
-                description: t("extensions:develop.emailAndSMS.description.description"),
-                pageTitle: t("extensions:develop.emailAndSMS.heading.heading"),
-                title: t("extensions:develop.emailAndSMS.title.heading")
-            };
-        } else if (isSMSProviderEnabled) {
-            return {
-                description: t("extensions:develop.emailAndSMS.description.onlySMSProvider"),
-                pageTitle: t("extensions:develop.emailAndSMS.heading.onlySMSProvider"),
-                title: t("extensions:develop.emailAndSMS.title.onlySMSProvider")
-            };
-        } else if (isEmailProviderEnabled) {
-            return {
-                description: t("extensions:develop.emailAndSMS.description.onlyEmailProvider"),
-                pageTitle: t("extensions:develop.emailAndSMS.heading.onlyEmailProvider"),
-                title: t("extensions:develop.emailAndSMS.title.onlyEmailProvider")
-            };
-        }
-    };
-
     return (
         <PageLayout
-            pageTitle={ getPageDetails().pageTitle }
-            title={ getPageDetails().title }
-            description={ getPageDetails().description }
+            pageTitle={ t("extensions:develop.notificationChannel.heading") }
+            title={ t("extensions:develop.notificationChannel.title") }
+            description={ t("extensions:develop.notificationChannel.description") }
             data-testid={ `${componentid}-page-layout` }
         >
-            { isEmailProviderEnabled && (
-                <>
-                    <Grid xs={ 12 } lg={ 6 }>
+            <Grid container spacing={ 2 }>
+                { featureConfig.emailProviders?.enabled
+                  && !(featureConfig?.emailProviders?.disabledFeatures?.includes("superTenantProvider"))
+                  && (
+                      <Grid xs={ 12 } md={ 4 } lg={ 4 } xl={ 4 }>
+                          <SettingsSection
+                              data-componentid={ "account-login-page-section" }
+                              data-testid={ "account-login-page-section" }
+                              description={ t("extensions:develop.emailProviders.description") }
+                              icon={
+                                  (
+                                      <EnvelopeIcon size="small" className="icon"/>
+                                  ) }
```

Comment:
Check whether we can keep this in a single-line
```suggestion
                              icon={ ( <EnvelopeIcon size="small" className="icon"/> ) }
```

Reply:
Fixed with df119ce